charts/seed-controlplane/charts/dependency-watchdog/templates/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: dependency-watchdog
+  namespace: {{ .Release.Namespace }}
+  labels:
+    role: dependency-watchdog
```

Comment:
Do we intentionally not add `garden.sapcloud.io/role: controlplane` and the health checks via https://github.com/gardener/gardener/blob/f4214897b6341954061f18d1cdddade86371eac2/pkg/operation/common/types.go#L571? Any thoughts @rfranzke?

Reply:
Good point, we should definitely add this now that this component is part of the control plane.